docs/source/tutorials/basic_renewal_model.qmd

```diff
@@ -214,7 +214,9 @@ model1.run(
     num_samples=1000,
     data_observed_infections=sim_data.observed_infections,
     rng_key=jax.random.PRNGKey(54),
-    mcmc_args=dict(progress_bar=False, num_chains=2, chain_method="sequential"),
+    mcmc_args=dict(
```

Comment:
Not for this PR but these diffs remind me that we need to get something working for shared formatting of `.qmd` files

Reply:
I have posted on Teams and updated the this issue in this repo https://github.com/CDCgov/multisignal-epi-inference/issues/159

Reply:
Thanks!